gallery/src/pages/StylingComponents.tsx

```diff
@@ -85,6 +85,48 @@ const StylingRSCPage = () =>
         visual state. These classes are common across all components that might need them, especially in the case of the
         validation utility classes.
       </p>
+
+      <h3 className="nx-h3 nx-tile__section-header">CSS Resources</h3>
+      <p className="nx-p">
+        Many components in the RSC are layed out using CSS Flexbox or CSS Grid. Flexbox is well
```

Comment:
Wouldn't it be "laid"?

Reply:
> Wouldn't it be "laid"?

I was curious so I did a bit of research and while "layed" has become a common alternate spelling "laid" is the correct past tense. https://github.com/sonatype/sonatype-react-shared-components/pull/308/commits/d92adcbbde5c560e9dbb4d8ce989930c2219fcb6